src/components/transport_manager/CMakeLists.txt

```diff
@@ -43,6 +43,12 @@ include_directories (
   ${LOG4CXX_INCLUDE_DIRECTORY}
 )
 
+if (CMAKE_SYSTEM_NAME STREQUAL "QNX")
```

Comment:
@LuxoftAKutsan 
Just note:
As for me private headers (in this case platform specific listeners does NOT belongs to component public interface) should be contained in src directory.
component 'include' dir should contain just public headers using by other components.
Such technique simplifies using library for clients - developers who just using transport manager in other components.
That was my small note that I maybe wants to discuss in future(according to task SDL improvements).

Reply:
@EKuliiev public headers that other components should have access to are in `components/include/{component}.`
Most of headers that are in `components/{component}/include/` are private and part of implementation.